### PR TITLE
Fix remove function for ControllableQueue

### PR DESCRIPTION
**Pull request by bf8086**
There is a bug in `ControllableQueue.remove` where it is trying to remove the item from priority queue which holds a list of wrappers. This change fixed it by comparing the value of the wrapper to the item instead.

### CODE REVIEW

File: core/common/src/main/java/alluxio/util/executor/ControllableQueue.java [hunk 1]
```diff
@@ -89,7 +89,7 @@ public T pop() {
    * @return true if the element removed successfully, false otherwise
    */
   public boolean remove(T element) {
-    return mQueue.remove(element);
+    return mQueue.removeIf(x -> x.mValue == element);
```
Comment (gpang) [inline core/common/src/main/java/alluxio/util/executor/ControllableQueue.java]: NIT: use `getValue()` instead of accessing private member?

Comment (bf8086) [reply core/common/src/main/java/alluxio/util/executor/ControllableQueue.java]: Done.

**Comment by bf8086** [comment]
Jenkins, test this please.

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Pull request merged**